hello-oboe/app/src/main/java/com/google/example/hellooboe/MainActivity.kt

```diff
@@ -44,7 +46,12 @@ class MainActivity : AppCompatActivity() {
 
     override fun onResume() {
         super.onResume()
-        createStream()
+        if (createStream() != 0) {
+            val errorString : String = getString(R.string.error_msg)
+//            "Error: unable to create Oboe stream \n please report issue @ https://github.com/google/oboe"
```

Comment:
nit: You can remove this comment, I think?

Reply:
good catch: done!  thx